src/main/kotlin/com/msg/gcms/domain/webhook/util/impl/CreateMessageUtilImpl.kt

```diff
@@ -0,0 +1,38 @@
+package com.msg.gcms.domain.webhook.util.impl
+
+import com.msg.gcms.domain.club.enums.ClubType
+import com.msg.gcms.domain.webhook.util.CreateMessageUtil
+import org.springframework.stereotype.Component
+import org.springframework.stereotype.Service
+
+@Component
+class CreateMessageUtilImpl : CreateMessageUtil {
+
+    override fun execute(club_name: String, club_type: ClubType, club_img: String) = """
```

Comment:
```suggestion
    override fun execute(clubName: String, clubType: ClubType, clubImg: String) = """
```
변수 표기법 lowerCamlCase로 일관성 맞춰주는게 좋을거같아요!